ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -257,6 +266,45 @@ where is_in_physics_bubble(vessel)
            has_active_vessel_in_space();
   }
 
+  private void OverrideRSASTarget(FlightCtrlState state) {
+    if (override_rsas_target_ && FlightGlobals.ActiveVessel.Autopilot.Enabled) {
+      FlightGlobals.ActiveVessel.Autopilot.RSAS.SetTargetOrientation(
+          rsas_target_,
+          reset_rsas_target_);
+    }
+    reset_rsas_target_ = false;
+  }
+
+  // Returns false if the file does not exist.
+  private bool LoadTextureIfExists(ref UnityEngine.Texture texture,
+                                   String path) {
+    string full_path =
+        KSPUtil.ApplicationRootPath + Path.DirectorySeparatorChar +
+        "GameData" + Path.DirectorySeparatorChar +
+        "Principia" + Path.DirectorySeparatorChar +
+        "assets" + Path.DirectorySeparatorChar +
+        path;
+    if (File.Exists(full_path)) {
+      var texture2d = new UnityEngine.Texture2D(2, 2);
+      bool success = texture2d.LoadImage(
+          File.ReadAllBytes(full_path));
+      if (!success) {
+        Log.Fatal("Failed to load texture " + path);
```

Comment:
Logging the full path, but the point here is that we don't want a hard error, it's for optional textures. If navball_compass.png is missing, we default to the stock navball. For the other textures we have a hard error provided by `LoadTextureOrDie`.